snow/consensus/snowball/nnary_snowball.go

```diff
@@ -9,7 +9,22 @@ import (
 	"github.com/ava-labs/avalanchego/ids"
 )
 
-var _ NnarySnowball = (*nnarySnowball)(nil)
+var (
+	_ NnarySnow        = (*nnarySnowball)(nil)
+	_ ConsensusFactory = SnowballFactory{}
```

Comment:
nit: fix this i/f check for consistency with the rest of the repo

Reply:
Ya, we can change this. I made it this way since it's just a value, but can definitely conform for style.